packages/microsoft_sentinel/manifest.yml

```
@@ -1,7 +1,7 @@
 format_version: 3.2.1
```

@seanrathier this is the correct format version, right?
```suggestion
format_version: 3.2.3
```

https://github.com/elastic/package-spec/blob/964c4a69e024cc464c4808720ba0db9f001a82a7/spec/changelog.yml#L46-L53

---

> @seanrathier this is the correct format version, right?

Yes, `3.2.3` is the most recent version that includes the `is_default` property. 

https://github.com/elastic/integrations/blob/main/packages/cloud_security_posture/manifest.yml#L110

---

version 3.2.3 also included the agentless agent resource options.

https://github.com/elastic/package-spec/blob/main/spec/integration/manifest.spec.yml#L211

---

Thank you! Updated to `3.2.3`